src/databricks/labs/lsql/deployment.py

```diff
@@ -1,46 +1,53 @@
-import datetime as dt
 import logging
 import pkgutil
 from typing import Any
 
-from databricks.sdk.errors import InternalError
-from databricks.sdk.retries import retried
-
 from databricks.labs.lsql.backends import Dataclass, SqlBackend
 
 logger = logging.getLogger(__name__)
 
 
 class SchemaDeployer:
-    def __init__(self, sql_backend: SqlBackend, inventory_schema: str, mod: Any):
+    """Deploy schema, tables, and views for a given inventory schema."""
+
+    def __init__(
+        self,
+        sql_backend: SqlBackend,
+        inventory_schema: str,
+        mod: Any,
+        *,
+        inventory_catalog: str = "hive_metastore",
```

Comment:
```suggestion
        catalog: str = "hive_metastore",
```
let's use new naming conventions for new feat